src/data/constants.ts

```diff
@@ -1,6 +1,16 @@
 export const VSCODE_PROJECT_CONFIG = {
   project: '<your-key-here>',
   sonarURL: '<your-sonar-url>',
+  auth:
+  {
+    username: '<sonar-username>',
+    password: '<sonar-password>'
+  }
+};
+
+export const VSCODE_PROJECT_JSON_FORMAT_OPTIONS = {
+  spaces: 4,
```

Comment:
Why is this needed ?

Reply:
Hi @pedroresende ,
I added this to format project.json file.
As previosly newly created project.json file have all keys and values in one line. After adding it, newly created project.json file will look:
{
    project: '<your-key-here>',
    sonarURL: '<your-sonar-url>',
    auth:
    {
      username: '<sonar-username>',
      password: '<sonar-password>',
    }
}